src/data/complex/exponential.lean

```diff
@@ -1510,6 +1523,71 @@ calc cos 2 = cos (2 * 1) : congr_arg cos (mul_one _).symm
           zero_le_two) _
   ... < 0 : by norm_num
 
+lemma exp_bound_div_one_sub_of_interval_approx  {x : ℝ} (h1 : 0 ≤ x) (h2 : x ≤ 1) :
+  ∑ (j : ℕ) in finset.range 3, x ^ j / (j.factorial)
+  + x ^ 3 * ((3 : ℕ) + 1) / ((3 : ℕ).factorial * (3 : ℕ))
+  ≤ ∑ j in (finset.range 3), x ^ j :=
+begin
+  repeat {rw finset.sum}, norm_num,
```

Comment:
```suggestion
  norm_num [finset.sum],
```

`repeat {rw...}` is basically a worse version of `simp only [...]`; `norm_num` is kind of like `simp`, so you can pass arguments to it too